### DISCUSSION TITLE
Logs are flooded with "statvfs '/run/user/1000/doc' failed: operation not permitted"

### DISCUSSION THREAD

**Discussion by nobara256** [Joshua-Riek/ubuntu-rockchip#136] [Q&A]
`[WARNING] statvfs /run/user/1000/doc failed: Operation not permitted` is flooding the logs, anyone know what is causing it? I am not sure if it was there before v1.14

**Comment by Joshua-Riek** [comment]
Are you using flatpack? I found this related issue. https://github.com/flatpak/flatpak/issues/3660

**Comment by nobara256** [comment]
I am not using flatpack. The problem seems be there with the stock install.

**Reply by Joshua-Riek** [reply]
Strange, I have not noticed this. Could you put the dmesg logs into a text file and upload them here?

**Reply by nobara256** [reply]
The warning seems to be coming from gnome system monitor. Makes sense as I keep system monitor open to track CPU spikes while running heavy tasks. I am unable to find the logs in dmesg but they appear under Application logs in the gnome log app. I can see the warnings under /var/log/syslog.

here is the last portion of syslog file:  [syslog.txt](https://github.com/Joshua-Riek/ubuntu-orange-pi5/files/11414203/syslog.txt)

**Reply by Joshua-Riek** [reply]
It does not seem fatal, but I can look later this week. I'm still investigating the strange GPU freeze issue.
